lti_consumer/tests/unit/test_api.py

```diff
@@ -291,12 +291,12 @@ def test_retrieve_with_block(self):
             {
                 'client_id': lti_config.lti_1p3_client_id,
                 'keyset_url': 'https://example.com/api/lti_consumer/v1/public_keysets/{}'.format(
-                    lti_config.location
+                    lti_config.config_id
                 ),
                 'deployment_id': '1',
                 'oidc_callback': 'https://example.com/api/lti_consumer/v1/launch/',
                 'token_url': 'https://example.com/api/lti_consumer/v1/token/{}'.format(
-                    lti_config.location
+                    lti_config.config_id
```

Comment:
Shouldn't we check both options (for backward compatibility)?

Reply:
Good catch here. Yes. We will need to. In fact, I will have to make sure these urls for the existing blocks aren't persisted (if it is not happening already). Otherwise we run into the situation where XBlock will show an URL with the `config_id` in the UI while it might be configured with `location` in the tool. Functionally it wouldn't break anything, but will be a confusing situation if someone tries to change/debug the integration for some reason.